docs/fr/index.md

```diff
@@ -1,25 +1,36 @@
 # Bienvenue
 
-Ceci est la documentation officielle de l'éditeur Zettlr Markdown. Cette documentation décrit les fonctionnalités offertes par Zettlr pour écrire du texte et organiser vos fichiers, ainsi que pour personnaliser votre expérience utilisateur. Si vous n'avez pas encore téléchargé Zettlr, vous pouvez le faire maintenant en vous rendant sur [notre page de téléchargement](https://www.zettlr.com/download). Zettlr est actuellement pris en charge sur Windows (7 ou plus récent), macOS (10.10 ou plus récent) et diverses distributions Linux. Veuillez consulter les [instructions de configuration](install.md) pour des instructions spécifiques à la plate-forme sur l'installation de l'application sur votre ordinateur.
+Ceci est la documentation officielle de Zettlr. Vous trouverez ici des descriptions détaillées de chaque fonctionnalité et comment vous pouvez les modifier en fonction de vos besoins. Consultez ce manuel chaque fois que vous êtes bloqué. Vous pouvez ouvrir ce manuel depuis l'application en appuyant sur « F1 » ou en cliquant sur l'élément de menu « Aide » &rarr; « Ouvrir le manuel d'utilisation » (connexion Internet requise).
 
-## Dans cette documentation
+!!! note
 
-Cette documentation est structurée en gardant à l'esprit le vaste ensemble de fonctionnalités et le groupe cible de Zettlr. Nous avons trié toutes les fonctionnalités disponibles dans quelques catégories centrales. Si vous savez exactement ce que vous recherchez, la barre de recherche en haut à droite est le bon point de départ pour vous, sinon, veuillez suivre les descriptions ci-dessous:
+    Cette documentation est disponible en plusieurs langues. Vous pouvez sélectionner une autre langue à partir de la [page de destination principale](https://docs.zettlr.com/). Veuillez noter que ces traductions peuvent être incomplètes. En cas de doute, consultez toujours cette version (anglaise) de la documentation, qui est régulièrement mise à jour.
```

Comment:
My French is not too good, but "consultez toujours cette version (anglaise)" sounds like a literal translation of the note of the English translation indicating that "this (the English)" is the main one – is this correct? If so, what it should read should be something along these lines:

> "If in doubt, consult the English documentation […]"

You know what I mean …?